website/docs/r/security_center_hybrid_vm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_hyrbid_vm_server_vulnerability_assessment"
+description: |-
+  Manages an Azure Vulnerability Assessment (Qualys) to an Azure ARC server.
+---
+
+# azurerm_security_center_hyrbid_vm_server_vulnerability_assessment
+
+Manages an Azure Server Vulnerability Assessment (Qualys) to a VM.
+
+-> **NOTE** Azure Defender has to be enabled on the subscription in order for this resource to work.
+See this [documentation](https://docs.microsoft.com/en-us/azure/security-center/security-center-get-started) to get started.
+
+## Example Usage
+
+```hcl
+resource "azurerm_security_center_hyrbid_vm_server_vulnerability_assessment" "example" {
+  hydrid_machine_id = "/hybrid/machine/id"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `hybrid_machine_id` - (Required) The ID of the Azure ARC server to be monitored by vulnerability assessment. Changing this forces a new resource to be created.
+
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the Vulnerability Assessment resource.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 5 minutes) Used when creating the Advanced Threat Protection.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Advanced Threat Protection.
+* `delete` - (Defaults to 10 minutes) Used when deleting the Advanced Threat Protection.
+
+## Import
+
+Server Vulnerability Assessments can be imported using the `resource id`, e.g.
```

Comment:
there's three different names listed here, can we standardize this on "Arc Virtual Machine Server Vulnerability Assessment"?